test/test_cpp_extensions_jit.py

```diff
@@ -365,19 +365,6 @@ def test_lenient_flag_handling_in_jit_extensions(self):
         z = module.tanh_add(x, y).cpu()
         self.assertEqual(z, x.tanh() + y.tanh())
 
-    def test_complex_registration(self):
```

Comment:
What was the original reason for this test? Are we sure we don't need it anymore?

Reply:
IT was originally to test that the complex out-of-tree extension point was working. But since Anjali is working on putting complex in tree we don't need this extension point anymore.